backend/src/main/resources/com/bakdata/conquery/resources/admin/ui/queries.html.ftl

```diff
@@ -0,0 +1,322 @@
+<#import "templates/template.html.ftl" as layout>
+<@layout.layout>
+<style>
+   .title{
+        font-size: 18px;
+        font-family: arial, sans-serif;
+   }
+
+   .inner{
+           font-size: 12px;
+           color: grey;
+   }
+   .innerBtn{
+               font-size: 12px;
+   }
+
+   .btn {
+        padding : 5px 5px;
+   }
+   .progress{
+     margin : 8px;
+   }
+   table {
+   font-family: arial, sans-serif;
+   border-collapse: collapse;
+   width: 100%;
+   }
+   td, th {
+   border: 1px solid white;
+   text-align: left;
+   padding: 8px;
+   }
+   tr:nth-child(even) {
+   background-color: #e6ffff;
+   }
+
+</style>
+<script type="text/javascript">
+var queries = [];
+var k = 0;
+var queryCounter = 0;
+
+var runningQueriesTable = {};
+var notStartedQueriesTable = {};
+var failedQueriesTable = {};
+var succeedQueriesTable = {};
+
+var runningNbrElt = {};
+var notStartedNbrElt = {};
+var failedNbrElt = {};
+var succeedNbrElt = {};
+window.onload = (event) => {
+    setInterval(getQueries, 10000);
+    runningQueriesTable = document.getElementById("runningQueriesTable");
+    notStartedQueriesTable = document.getElementById("notStartedQueriesTable");
+    failedQueriesTable = document.getElementById("failedQueriesTable");
+    succeedQueriesTable = document.getElementById("succeedQueriesTable");
+
+    runningNbrElt = document.getElementById("runningNbr");
+    notStartedNbrElt = document.getElementById("notStartedNbr");
+    failedNbrElt = document.getElementById("failedNbr");
+    succeedNbrElt = document.getElementById("succeedNbr");
+
+
+}
+
+function updateQueriesTable() {
```

Comment:
Hier ähnlich in in jobs.html.ftl bitte auch einen Pause Knopf einbauen (und schlauer als ich oben statt ganz unten platzieren)